### PR TITLE
Add people files

### PR DESCRIPTION
**Pull request by scott-ace-newton**

### CODE REVIEW

File: reader.go [hunk 1]
```diff
@@ -51,8 +51,11 @@ func (sfr *FactsetReader) Read(fRes factsetResource, dest string) (string, error
 	if err != nil {
 		return lastVers, err
 	}
+	factsetFiles := strings.Split(fRes.fileNames, ";")
+	for _, factsetFile := range factsetFiles {
+		err = sfr.unzip(lastVers, factsetFile, dest)
```
Comment (ghost) [inline reader.go]: Only the value of `err` for the last file will be used, others will be masked.  This should probably return on the first non-nil error.

File: service.go [hunk 2]
```diff
@@ -61,26 +62,34 @@ func (s service) fetchResource(res factsetResource) error {
...
-	}()
+	factsetFiles := strings.Split(res.fileNames, ";")
 
-	log.Infof("Resource [%s] was succesfully read from Factset in %d", fileName, time.Since(start))
+	for _, factsetFile := range factsetFiles {
+		extension := filepath.Ext(factsetFile)
+		nameWithoutExt := strings.TrimSuffix(factsetFile, extension)
+		fileNameOnS3 := nameWithoutExt + "_" + fullVersion + extension
+
+		log.Infof("Resource [%s] was succesfully read from Factset in %d", fileName, time.Since(start).String())
```
Comment (ghost) [inline service.go]: Won't this return a increasingly large time for each file?

**Review by ghost — CHANGES_REQUESTED** [review]

File: reader.go (hunk 1 above)
Comment (scott-ace-newton) [reply reader.go]: Good point

File: service.go (hunk 2 above)
Comment (scott-ace-newton) [reply service.go]: Well it kicks off the download of all files as soon as its triggered, so yes. Im not really sure what you mean?

File: reader.go (hunk 1 above)
Comment (scott-ace-newton) [reply reader.go]: Fixed

**Pull request closed without merging**